Assignment

```diff
@@ -1 +1,83 @@
+import csv
 
+def read_weather_data(weather_data):
+    weather_data = []
+    with open( r"C:\Users\ue\Desktop\weatherAUS\weatherAUS.csv") as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            weather_data.append(row)
+        return weather_data
+def calculate_avg_temperatures(weather_data):
```

Comment:
@hareem211 Study what is main function in python code files and what's purpose of that function?

Reply:
In Python, the role of main function is to act as the starting point of execution for any software program. The execution of the program starts only when the main function is defined in python because the program executes only when it runs directly.